explorer/client/src/__tests__/reducers/adminOperatorsIndex.test.ts

```diff
@@ -17,26 +21,35 @@ describe('reducers/adminOperatorsIndex', () => {
     expect(state.adminOperatorsIndex).toEqual(INITIAL_STATE.adminOperatorsIndex)
   })
 
-  describe('FETCH_ADMIN_OPERATORS_SUCCEEDED', () => {
-    it('can replace items', () => {
-      const action: FetchAdminOperatorsSucceededAction = {
-        type: 'FETCH_ADMIN_OPERATORS_SUCCEEDED',
-        data: {
-          chainlinkNodes: [],
-          meta: {
-            currentPageOperators: {
-              data: [{ id: '9b7d791a-9a1f-4c55-a6be-b4231cf9fd4e' }],
-              meta: { count: 100 },
-            },
+  it('FETCH_ADMIN_OPERATORS_SUCCEEDED can replace items', () => {
+    const action: FetchAdminOperatorsSucceededAction = {
+      type: 'FETCH_ADMIN_OPERATORS_SUCCEEDED',
+      data: {
+        chainlinkNodes: [],
+        meta: {
+          currentPageOperators: {
+            data: [{ id: '9b7d791a-9a1f-4c55-a6be-b4231cf9fd4e' }],
+            meta: { count: 100 },
           },
         },
-      }
-      const state = reducer(INITIAL_STATE, action)
+      },
+    }
+    const state = reducer(INITIAL_STATE, action)
+
+    expect(state.adminOperatorsIndex.items).toEqual([
+      '9b7d791a-9a1f-4c55-a6be-b4231cf9fd4e',
+    ])
+    expect(state.adminOperatorsIndex.count).toEqual(100)
+    expect(state.adminOperatorsIndex.loaded).toEqual(true)
+  })
 
-      expect(state.adminOperatorsIndex).toEqual({
-        items: ['9b7d791a-9a1f-4c55-a6be-b4231cf9fd4e'],
-        count: 100,
-      })
+  it('FETCH_ADMIN_OPERATORS_ERROR sets loaded', () => {
+    const action = partialAsFull<FetchAdminOperatorsErrorAction>({
+      type: 'FETCH_ADMIN_OPERATORS_ERROR',
+      error: new Error(),
     })
+    const state = reducer(INITIAL_STATE, action)
+
+    expect(state.adminOperatorsIndex.loaded).toEqual(true)
```

Comment:
Should we also check that an `errMsg` was set?

Reply:
That check is handled in the `<Table />` with the combination of `loaded` + records